components/org.wso2.carbon.identity.application.authenticator.google/src/test/java/org/wso2/carbon/identity/application/authenticator/google/UtilsTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.authenticator.google;
+
+import junit.framework.TestCase;
+import org.wso2.carbon.identity.application.authentication.framework.exception.AuthenticationFailedException;
+
+import java.io.File;
+import java.util.Scanner;
+
+public class UtilsTest extends TestCase {
+
+    // Token is fed by a text file
+    String token;
+    String audience = "132708446238-d3lpgkp0t8fiko20ii217lmbdftnojei.apps.googleusercontent.com";
+
+    public void setUp() throws Exception {
+
+        super.setUp();
+        File file = new File(
+                "/Users/indeewariwijesiri/Documents/Servers/1.0.4/identity-outbound-auth-google/components/org.wso2.carbon.identity.application.authenticator.google/src/test/resources/token.txt");
```

Comment:
Let's add this file to a test resource folder and read from there.

Reply:
Fixed 3335ebd823fbdae6364b8cc712db7975815b8ee6